inc/sainexthopgroup.h

```diff
@@ -184,6 +184,19 @@ typedef enum _sai_next_hop_group_attr_t
      */
     SAI_NEXT_HOP_GROUP_ATTR_SELECTION_MAP,
 
+    /**
+     * @brief Hierarchical next hop group.
+     *
+     * false: Control plane will set this flag as false when Nexthop group consists of only IP nexthop
+     * true: Control plane will set this flag as true when nexthop group consists of both tunnel and IP nexthop
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default true
+     * @validonly SAI_NEXT_HOP_GROUP_ATTR_TYPE == SAI_NEXT_HOP_GROUP_TYPE_DYNAMIC_UNORDERED_ECMP
+     */
+    SAI_NEXT_HOP_GROUP_ATTR_HIEARCHICAL_NEXTHOP,
+
```

Comment:
I was considering that this should be @isresource for resource-monitoring. Though this could be a follow-up, as it only solves CRM for group count across multiple levels, but not members